csm/cli/support_bundle/csm_bundle_generate.py

```diff
@@ -137,27 +137,27 @@ def generate_bundle(args):
         GenerateCsmBundle.stacktrace = args[const.SB_STACKTRACE]
         target_path = os.path.join(GenerateCsmBundle.target_path, const.CSM_COMPONENT_NAME)
         os.makedirs(target_path,exist_ok=True)
-
+        # Apply Time filter on CSM Logs Default: P5d
+        csm_time_filtered_logs_dir = os.path.join(const.CSM_SETUP_LOG_DIR,
+            f"{const.CSM_COMPONENT_NAME}_logs_time")
```

Comment:
1. Create 1 folder name /tmp/CSM_filtered_logs
2. Do time filtering. 
3. Created a new folder /tmp/CSM_filtered_logs/size
4. Input folder step 1 and output folder step 3